docs/studio.rst

```diff
@@ -0,0 +1,94 @@
+Project overview
+----------------
+
+Once you have created a project, you can find it via the sidebar link Projects. Here you will find the list of all your projects. When inside a project you can see the following tabs in the sidebar:
+
+1. **Dashboard**: The dashboard provides an overview of the project. The controller and combiner(s) are listed under "Network". This is also where you can find the current FEDn version and get the option to upgrade to a newer version if available
+2. **Clients**: management of client configurations and a list of current clients. Observe that this feature does not deploy clients, instead it configures a client config that contains a unique token which is required to connect to the reducer and the combiner
+3. **Combiners**: a list of combiners. Observe number of active clients for each combiner
+4. **Sessions**: a list of sessions with related models. Configure and start a new session. Upload compute package and seed model, set number of rounds, timeout limit etc
+5. **Models**: a list of models generated across sessions
+6. **Events**: a log of events from the combiner and the clients of the federated network
+7. **Settings**: project settings, including the option to delete the project and give access to other users
+
+.. image:: img/studio_project_overview.png
+
+
+Configure and attach clients
+----------------------------
+
+This step is very similar to how a client is connected to a local FEDn instance (:doc:`quickstart`). The main difference is that the client needs an authorization token to connect to a federation hosted on FEDn Studio. These are the steps to configure and attach a client to a project:
+
+In order to set up the local environment and configure local client data access, on your local machine, navigate to the mnist-pytorch example:
+
+
+.. code-block:: bash
+
+    cd fedn/examples/mnist-pytorch
+
+Generate files
+~~~~~~~~~~~~~~
+
+Run the following commands:
+
+.. code-block:: bash
+
+    bin/init_venv.sh
+    bin/get_data
+    bin/split_data
+    bin/build.sh
+
+These commands will generate training data and two files: package.tgz and seed.npz. The first is a compute package file and the second is a model file that can be used as the inital seed model. (For a more detailed explanation of the commands, see the :doc:`quickstart`)
+
+Upload files
+~~~~~~~~~~~~
+
+In the Studio UI, navigate to the project you created and click on the "Sessions" tab. Click on the "New Session" button. Under the Compute package tab, select a name and upload the generated package file. Under the Seed model tab upload the generated seed file:
+
+.. image:: img/upload_package.png
+
+Connect client
+~~~~~~~~~~~~~~
+
+Navigate to clients and click on the "Download config" button. This downloads a client config file. Place this file in the mnist-pytorch directory on your local machine. Rename the file to client.yaml. Then edit it to set the name of the client.
+
+Later, in Studio, click on the "Connect client" button.
```

Comment:
I think we should update this with "native instructions" also.

Reply:
Would like to leave this as is for now... Just until we've settled on a client connect view in studio.